src/chocolatey/infrastructure.app/nuget/NuGetEndpointResources.cs

```diff
@@ -153,7 +153,33 @@ private T ResolveResource<T>()
             {
                 if (!_resolvingFailed)
                 {
-                    this.Log().Warn(ex.InnerException.Message);
+                    // Unwrap the AggregateException as its surface message is useless
+                    Exception error = ex.InnerException;
+                    this.Log().Warn(error.Message);
+
+                    // Enumerate the inner exceptions, log all but the last one in the list to debug
+                    string message = null;
+                    foreach (var err in error.InnerException.Enumerate())

```

Comment:
```suggestion
                    this.Log().Warn(error.Message);
                    // Enumerate the inner exceptions, log all but the last one in the list to debug
                    string message = null;

                    foreach (var err in error.InnerException.Enumerate())
```
This is more in line with the coding style right? Although, I feel like the other one is more readable...

Reply:
The more I look at this, the more I don't like my suggestion here. The string is very much a part of the foreach loop, and the comment is the start of it. Disregard this requested change 👍